release-schema.json

```
@@ -208,6 +208,11 @@
           "title": "Contract period",
           "description": "The period over which the contract is estimated or specified to be active. If the lot does not specify explicit dates, the duration field can be used.",
           "$ref": "#/definitions/Period"
+        },
+        "buyer": {
+          "title": "Buyer",
+          "description": "A buyer is an entity whose budget will be used to pay for goods, works or services related to this particular lot. This may be different from the procuring entity who may be specified in the tender data.",
```

Use 'this lot' rather than 'this particular lot' to align with the descriptions of other properties of `Lot`.

Use 'in' rather than 'related to' since items belong to lots.

```suggestion
          "description": "A buyer is an entity whose budget will be used to pay for the goods, works or services in this lot. This may be different from the procuring entity who may be specified in the tender data.",
```